README.md

```diff
@@ -75,7 +75,7 @@ following keys:
     (defaults to session length).
   - `httpOnly` - flags the cookie to be accessible only by the web server
     (defaults to `false`).
-  - `sameSite` - sets the same site policy for the cookie (defaults to none).
+  - `sameSite` - sets the same site policy for the cookie (defaults to `none`). Note, that from chrome version 80 it will be `lax` by [default](https://chromestatus.com/feature/5088147346030592).
```

Comment:
I think this is incorrect, though, right? I see no changes made here that would default it to `none`? Can you clarify on this?

Reply:
Yeap, if you go through the link there will be info, that chrome browser is setting on sameSite flag to enabled by default from version 80. So, right now it's not enabled in browser by default. If I correctly understand, it's the same behaviour with none option, isn't?

Reply:
The new cookies adds none support. But it has to be set. This reads like it's set here but nothing in this module is setting it? For example, what does the `set-cookie` module look like that this module makes when using the default settings on your branch?

Reply:
Oh got it. You mean, I need to delete old wrong definition.